cmake/Platform.cmake

```diff
@@ -20,8 +20,9 @@ elseif(VCPKG_TARGET_TRIPLET STREQUAL "x86-android")
 endif()
 
 if(ANDROID_ABI)
-  set(ANDROID_NDK_VERSION "$ENV{ANDROID_NDK_HOME}" CACHE STRING "Android NDK version")
-  set(ANDROID_BUILD_TOOLS_VERSION "30.0.3" CACHE STRING "Android build-tools version")
+  set(ANDROID_NDK_VERSION "$ENV{ANDROID_NDK_VERSION}" CACHE STRING "Android NDK version")
+  set(ANDROID_NDK_HOME "$ENV{ANDROID_NDK_HOME}" CACHE STRING "Android NDK home path")
```

Comment:
Do we need this?

Reply:
I don't think so, but not sure we should get rid of it. ATM this change merely fix what was clearly a wrong value.

Reply:
I think If we don't need them we should get rid of them.

Reply:
@m-kuhn , :fearful: :wink: